app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -498,13 +503,32 @@ function AcuantCapture(
     }
   }
 
+  function onSelfieCaptureOpen() {
+    trackEvent('idv_sdk_selfie_image_capture_opened');
+
+    setIsCapturingEnvironment(true);
+  }
+
+  function onSelfieCaptureClosed() {
+    trackEvent('idv_sdk_selfie_image_capture_closed_without_photo');
+
+    setIsCapturingEnvironment(false);
+  }
+
   function onSelfieCaptureSuccess({ image }: { image: string }) {
+    trackEvent('idv_sdk_selfie_image_added', { attempt });
+
     onChangeAndResetError(image);
     onResetFailedCaptureAttempts();
     setIsCapturingEnvironment(false);
   }
 
-  function onSelfieCaptureFailure() {
+  function onSelfieCaptureFailure(error) {
+    trackEvent('idv_sdk_selfie_image_capture_failed', {
+      sdk_error_code: error.code,
+      sdk_error_message: error.message,
+    });
```

Comment:
I thought it would be useful to have the error code and error message at top level for easy log searching (as opposed to down a layer so that you'd need to look at `error.code` or `error.message`.

I also wanted to name them this way `sdk_error_...` so that it was clear these codes / messages were coming from the SDK and aren't necessarily what our code controls or what we're showing to the user intentionally (the SDK may be showing them without us controlling it).

@daviddsilvanava since you often are our top analytics detective, I thought I'd tag you here just to see if my guesses match your ideal logging structure.

This would make it so that you could hopefully search with a query more like:

```
| filter properties.event_properties.sdk_error_code = 2
```

instead of

```
| filter properties.event_properties.error.code = 2
```

One level out and I think the property name is a little more clear. But if you feel differently or have a different idea, I'm happy to update it!

Reply:
This works for me! Thanks!

Reply:
Thanks for taking a look!